src/rgw/store/dbstore/tests/CMakeLists.txt

```diff
@@ -10,4 +10,6 @@ include_directories(${CMAKE_INCLUDE_DIR})
 
 add_executable(unittest_dbstore_tests ${dbstore_tests_srcs})
 target_link_libraries(unittest_dbstore_tests ${CMAKE_LINK_LIBRARIES})
-add_ceph_unittest(unittest_dbstore_tests)
```

Comment:
'add_ceph_unittest' will add these tests to be run as part of 'make check' target.  Please include both the tests to it.

Reply:
> 

Sure, I thought it was weird to build the tests that way, but as the README.md said to build them that way confused me.